mediathread/djangosherd/models.py

```diff
@@ -235,12 +235,15 @@ def fully_qualify_references(self, text, host):
         # annotations
         regex_string = \
             r'(name=|href=|openCitation\()[\'"]/asset/(\d+)/annotations/(\d+)'
-        replace_with = r'href="http://%s/asset/\2/annotations/\3' % host
+        replace_with = \
+            r'href="http://%s/course/%s/react/asset/\2/annotations/\3' % (
+                host, course.id)
         text = re.sub(regex_string, replace_with, text)
 
         # assets
         regex_string = r'(name=|href=|openCitation\()[\'"]/asset/(\d+)/[^a]'
-        replace_with = r'href="http://%s/asset/\2/"' % host
+        replace_with = \
+            r'href="http://%s/course/%s/react/asset/\2/"' % (host, course.id)
```

Comment:
These regexes could probably be updated to https as well, but I'll merge this for now.